webapp/src/components/backstage/playbook_runs/playbook_run/checklists.tsx

```diff
@@ -7,15 +7,18 @@ import styled from 'styled-components';
 import {PlaybookRun} from 'src/types/playbook_run';
 
 import RHSChecklistList, {ChecklistParent} from 'src/components/rhs/rhs_checklist_list';
+import {Role} from '../shared';
 interface Props {
```

Comment:
This is a personal preference, but I'd rather have absolute pahts in the imports. Also, new line missing:
```suggestion
import {Role} from '../shared';

interface Props {
```